docs/content/how-to/upgrades.md

```diff
@@ -23,3 +23,67 @@ The HostedControlPlane orchestrates the rollout of the new version of the Contro
 `.spec.release` dictates the version of any particular NodePool.
 
 A NodePool will perform a Replace/InPlace rolling upgrade according to `.spec.management.upgradeType`.
+
```

Comment:
Let this be a ref to a section in Automated Machine Management, e.g NodePool "Upgrades" after this gets merged https://github.com/openshift/hypershift/pull/1804#discussion_r996708935